frontend/src/features/admin-form/settings/SettingsEndPage.tsx

```diff
@@ -0,0 +1,19 @@
+import { Skeleton } from '@chakra-ui/react'
+
+import { useAdminForm } from '../common/queries'
+
+import { CategoryHeader } from './components/CategoryHeader'
+import { EndPageSettingsInput } from './components/EndPageSettingsSection/EndPageSettingsInput'
+
+export const SettingsEndPage = (): JSX.Element => {
+  const { data: form, isLoading } = useAdminForm()
+
+  return (
+    <>
+      <CategoryHeader mb={0}>Customise Thank You page</CategoryHeader>
+      <Skeleton isLoaded={!isLoading && !!form}>
+        {form ? <EndPageSettingsInput {...form.endPage} /> : null}
```

Comment:
Ahh yes, skeleton won't appear if form isn't loaded. I didn't want to use `useAdminForm` in `EndPageSettingsInput` and inline skeleton states because I didn't want to deal with `undefined` `form` in `EndPageSettingsInput`, thought it would be simpler to deal with the loading in a parent component and assume that `endPage` is defined in `EndPageSettingsInput`